move-mutator/src/compiler.rs

```diff
@@ -339,10 +345,13 @@ pub(crate) fn compile_package(
 ) -> anyhow::Result<CompiledPackage> {
     let mut compilation_msg = vec![];
     let external_checks = vec![];
+    let resolved_graph = build_config
+        .clone()
```

Comment:
nit: I think we are safe to make this `build_config` mutable in the argument list and then we can avoid this clone
```suggestion
    let resolved_graph = build_config
```

Reply:
Resolving, cus you're on vacation. Ping me if I'm wrong and it needs changing.